SEED-platform/seed
# feat: updated units and added unit multiplier

#### Any background context you want to provide?
see ticket

#### What's this PR do?
Changes the units for one BETTER import and adds the ability to scale units before adding them to extra_data.

#### How should this be manually tested?
Run a BETTER analysis and check that GHG Emissions Reduction is reported in MTCO2e and matches that in the HTML report.

#### What are the relevant tickets?
https://github.com/SEED-platform/seed/issues/2843

#### Screenshots (if appropriate)
<img width="693" alt="Screen Shot 2022-03-17 at 12 48 27 AM" src="https://user-images.githubusercontent.com/564231/158753276-22058073-112a-423a-8192-fdec1abcdb54.png">

<img width="686" alt="Screen Shot 2022-03-17 at 12 48 34 AM" src="https://user-images.githubusercontent.com/564231/158753273-59f10b7f-ad90-4980-912b-8ea5776e9f00.png">
